plugins/listblock/plugin.js

```diff
@@ -104,6 +105,10 @@ CKEDITOR.plugins.add( 'listblock', {
 						text: html || value
 					};
 
+					if ( language ) {
+						data.language = language;
+					}
```

Comment:
Probably the logic that fallbacks to the editor's language should be moved here to always output the correct `[lang]` attribute in various toolbar buttons.